content/tutorials/portenta-h7/por-ard-ap/content.md

```diff
@@ -0,0 +1,131 @@
+
+# Portenta H7 as a WiFi Access Point
+Portenta H7 comes with an on-board WiFi and a bluetooth Module that allows to develop IoT applications that require wireless connectivity and Internet access. Turning the board into an access point allows it to create a WiFi network on its own and allows other devices to connect to it. In  this tutorial you will learn to set up your board as an access point web server and remotely control the red, green and blue LEDs on the built-in RGB LED by accessing an HTML page on your mobile device’s browser. 
+
+## What you will learn
+-   About the built-in WiFi + Bluetooth module.
+-   How a  client-server model works 
+-   How to create an HTTP communication channel between the board and an external device. 
+
+## Required hardware and software
+-   Portenta H7 board (<https://store.arduino.cc/portenta-h7>)
+-   One USB C cable (either USB A to USB C or USB C to USB C)
+-   Arduino IDE 1.8.10+  or Arduino Pro IDE 0.0.4 +
+-   A smart phone
+
+# Access Point Configuration
+The Portenta H7 features a  [Murata 1DX](https://wireless.murata.com/type-1dx.html), which is  a high performance chipset which supports  WiFi 802.11b/g/n + Bluetooth 5.1 BR/EDR/LE up to 65Mbps PHY data rate on WiFi and 3Mbps PHY data rate on Bluetooth. This module helps to configure the Portenta into three different modes of operation -  an Access Point,  a Station, or both. In this tutorial we will only focus on the access point configuration.
+
+When the  board is configured to operate as an access point, it can create its own wireless LAN ( WLAN ) network. In this mode, the board transmits and receives signals at 2.4 GHz allowing other electronic devices with WiFi capabilities using the same bandwidth to connect to the board.
+
+With the access point set up you create a client server architecture where the board provides a web server communicating with the client devices over HTTP. The connected devices can then make HTTP GET requests to the server to retrieve web pages served by the web server on the board. This makes the Portenta H7 an ideal board for developing IoT solutions where external client devices can send and receive information while more complex processing tasks take place on the server.
+
+![A client device communicating with the Portenta H7 through HTTP ](assets/por_ard_ap_tutorial_core_topic.svg?sanitize=true)
```

Comment:
@Lenardgeorge I'd suggest to use the same number "badge" with the circle around as we use in the other image.